src/modules/itemCounter.js

```diff
@@ -0,0 +1,14 @@
+const API_URL = 'https://www.themealdb.com/api/json/v1/1/filter.php?a=British';
+const itemCounter = document.querySelector('.item-counter');
+
+const menuCounter = () => {
+  fetch(API_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      const meal = data.meals;
+      const count = meal.length;
+      itemCounter.innerHTML = `(${count})`;
+    });
```

Comment:
- [ ] according to project requirements kindly use async and await with the fetch method to send and receive data.

Reply:
done thank you